uwblueprint/jump-math
# Admin Users Query

## Notion ticket link

[Admin Users Query] https://www.notion.so/uwblueprintexecs/47bf4727a2444bccad4a91952898b8f3?v=9658436bf9be409e90581631a7155f16&p=5b06976a90ca44a4b4393c3e59bcf0b2&pm=s

error state
<img width="750" alt="Screen Shot 2022-11-16 at 11 27 36 PM" src="https://user-images.githubusercontent.com/82241706/202355407-274d13fa-48d1-4f6a-825b-d8c138b1bffa.png">

loading state
<img width="505" alt="Screen Shot 2022-11-16 at 11 27 11 PM" src="https://user-images.githubusercontent.com/82241706/202355433-f85b3dcf-3429-49c8-88f5-16adb36ba432.png">




## Implementation description
* created the query on TeacherPage



## Steps to test
1. Go to TeacherPage.tsx, add the line "console.log(adminUsers)"
2. Navigate to localhost:3000/teachers, view the console, and verify the array is in the format
[{firstName, lastName, email}]




## What should reviewers focus on?
* query is formatted properly, handles everything required


## Checklist
